<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>0</y>
      <width>521</width>
      <height>481</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
     <widget class="QFrame" name="frame">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>50</y>
        <width>451</width>
        <height>251</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QFormLayout" name="formLayout">
       <item row="5" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Voltaje:</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QLineEdit" name="insert_voltaje"/>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Posicion en x:</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QSpinBox" name="pos_x"/>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Posicion en y:</string>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="QSpinBox" name="pos_y">
         <property name="maximum">
          <number>99</number>
         </property>
        </widget>
       </item>
       <item row="10" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>RED</string>
         </property>
        </widget>
       </item>
       <item row="10" column="1">
        <widget class="QSlider" name="set_red">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="12" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>GREEN</string>
         </property>
        </widget>
       </item>
       <item row="12" column="1">
        <widget class="QSlider" name="set_green">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="14" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>BLUE</string>
         </property>
        </widget>
       </item>
       <item row="14" column="1">
        <widget class="QSlider" name="set_blue">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>ID:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="insert_id"/>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="agregarInicio">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>370</y>
        <width>91</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Agregar inicio</string>
      </property>
     </widget>
     <widget class="QPushButton" name="AgregarFinal">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>370</y>
        <width>91</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Agregar final</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
     <widget class="QPlainTextEdit" name="plainTextEdit">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>471</width>
        <height>251</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="mostrar">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>300</y>
        <width>131</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Mostrar Neuronas</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Guardarlista">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>270</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Guardar</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Leerlista">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>270</y>
        <width>81</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Leer</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Page</string>
     </attribute>
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>471</width>
        <height>311</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="buscarNeurona">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>340</y>
        <width>191</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Buscar neurona</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>350</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Page</string>
     </attribute>
     <widget class="QGraphicsView" name="graphicsView">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>461</width>
        <height>311</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="Dibujar">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>350</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Dibujar neuronas</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
